MODULE.bazel

```diff
@@ -304,5 +304,5 @@ use_repo(
     "org_golang_x_tools",
 )
 
-bazel_dep(name = "rules_python", version = "1.5.3", dev_dependency = True)
-bazel_dep(name = "rules_shell", version = "0.5.1", dev_dependency = True)
+bazel_dep(name = "rules_python", version = "1.6.0-rc0", dev_dependency = True)
```

Comment:
The upgrade to `-rc0` cought my attention. It looks like tags [1.6.0](https://github.com/bazel-contrib/rules_python/releases/tag/1.6.0) and [1.6.0-rc0](https://github.com/bazel-contrib/rules_python/releases/tag/1.6.0-rc0) point to the same commits, but that's only their house-keeping method -in case of rc1 the tag of 1.6.0 would point to rc1. 
It seems however like they're going have a final release shortly https://github.com/bazel-contrib/rules_python/issues/3188 
Since that's only a dev_dependency it should be fine, or we can simply stick to last stable release 1.5.4

Reply:
Huh, I didn't realize they manage their tags like that. I'd definitely prefer a more official release, but [only 1.6.0-rc0 is available on the BCR right now](https://registry.bazel.build/modules/rules_python) and [supports aarch64-pc-windows-msvc builds](https://github.com/bazel-contrib/rules_python/blob/1.6.0-rc0/python/versions.bzl#L482).

In contrast, [1.5.4 doesn't support aarch64-pc-windows-msvc builds](https://github.com/bazel-contrib/rules_python/blob/1.5.4/python/versions.bzl). Like you said, since this is a `dev_dependency`, I felt like using a release candidate is OK, since it'll enable testing and debugging on Windows Arm64 sooner than later.